### PR TITLE
fix travis without installing opencga: copy conf into resources

### PR DESCRIPTION
**Pull request by jmmut**
This pr fixes your pr ebivariation/eva-pipeline#113

just fast-forward your develop branch to include this commit and your pr will pass travis

here there is my travis passing with this commit https://travis-ci.org/jmmut/eva-pipeline/builds/223621704

### CODE REVIEW

File: install-dependencies.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+
+mkdir opencga
+cp -r src/main/resources/opencga/conf opencga/
```
Comment (pabarcgar) [inline install-dependencies.sh]: Is OPEN_CGA env variable not available here?

**Review by pabarcgar — COMMENTED** [review]
just a small change

File: install-dependencies.sh (hunk 1 above)
Comment (jmmut) [reply install-dependencies.sh]: I'll try

Comment (jmmut) [reply install-dependencies.sh]: https://travis-ci.org/jmmut/eva-pipeline/builds/223873057

**Pull request merged**